src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1415,7 +1434,7 @@ private long getLaunchTimeoutMillis() {
      * @return maxNumRetries
      */
     public Integer getMaxNumRetries() {
```

Comment:
Are you stuck with API compatibility here, or can you change these getters to use an `int` instead? Or add `@NonNull` to them at least? :)

Reply:
done